independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/InjectionPointInfo.java

```diff
@@ -134,6 +134,13 @@ method, position, contains(paramAnnotations, DotNames.TRANSIENT_REFERENCE),
         this.position = position;
         this.isTransientReference = isTransientReference;
         this.isDelegate = isDelegate;
+
+        // validation - Event injection point can never be a raw type
+        if (DotNames.EVENT.equals(requiredType.name()) && requiredType.kind() == Type.Kind.CLASS) {
+            throw new DefinitionException(
+                    "Event injection point can never be raw type - please specify the type parameter. Bean: "
```

Comment:
The target does not have to be a bean...

Reply:
`sed 's/Bean/Injection point/'`